src/components/Notifications/NotificationTypes/Marketplace/BidAcceptedNotification.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react"
+import { Network } from "@dcl/schemas"
+import { useTheme } from "@mui/material"
+import { i18n } from "./BidAcceptedNotification.i18n"
+import { BidAcceptedIcon } from "../../../Icon"
+import { Mana } from "../../../Mana"
+import { NotificationItem } from "../../NotificationItem"
+import {
+  NotificationItemDescription,
+  NotificationItemTitle,
+} from "../../NotificationItem.styled"
+import { NotificationItemNFTLink } from "../../NotificationItemNFTLink"
+import {
+  BidAcceptedNotificationProps,
+  CommonNotificationProps,
+} from "../../types"
+import { formatMana, getBGColorByRarity } from "../../utils"
+
+const BidAcceptedNotification = React.memo(
+  (props: CommonNotificationProps<BidAcceptedNotificationProps>) => {
+    const { notification, locale } = props
+    const theme = useTheme()
+    return (
+      <NotificationItem
+        image={{
+          image: notification.metadata.image,
+          backgroundColor: getBGColorByRarity(notification.metadata.rarity),
+          icon: <BidAcceptedIcon />,
+        }}
```

Comment:
I think we might need to look a way to re-define this prop, as we have an image inside an image.